packages/create-svelte/bin.js

```diff
@@ -21,16 +21,41 @@ async function main() {
 	console.log(gray(`\ncreate-svelte version ${version}`));
 	console.log(disclaimer);
 
-	let cwd = process.argv[2] || '.';
-
-	if (cwd === '.') {
-		const opts = await prompts([
+	const package_name = await prompts(
+		[
 			{
 				type: 'text',
-				name: 'dir',
-				message: 'Where should we create your project?\n  (leave blank to use current directory)'
+				name: 'packagename',
+				message: 'What would you like to name your project?',
+				validate: (value) => {
+					const regex = new RegExp(/^(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/);
```

Comment:
~  is no longer allowed for new packages, see https://github.com/npm/validate-npm-package-name

Reply:
Interesting! That's probably a second bug fix for us -- the current function we're using to convert folder names to package names is still allowing them. I'll likely change this implementation to use the library you linked in your top level comment anyway -- I didn't realize that was an official NPM library when I saw it!